Docs: Better description for LocationScouts (#2674)
ArchipelagoMW/Archipelago: docs/network protocol.md

| locations | list\[int\] | The ids of the locations checked by the client. May contain any number of checks, even ones sent before; duplicates do not cause issues with the Archipelago server. |

### LocationScouts

> Sent to the server to retrieve the items that are on a specified list of locations. The server will respond with a [LocationInfo](#LocationInfo) packet with the items located in the scouted locations.

Reviewer: You could change this word to remove the double "with"
```suggestion
Sent to the server to retrieve the items that are on a specified list of locations. The server will respond with a [LocationInfo](#LocationInfo) packet containing the items located in the scouted locations.
```

> Fully remote clients without a patch file may use this to "place" items onto the in-game locations, such as displaying their names, items classification etc. before/upon pickup.

Reviewer 2: This description of this use case feels almost too specific

Author: You think so? What do you think is missed?

Author: I can't really think of sth better so I might need a suggestion on this one

Author: You think it'd be better if it just said "or" instead of "such as"?

> Fully remote clients without a patch file may use this to "place" items onto the in-game locations, such as displaying their names, items classification etc. before/upon pickup.

Reviewer: Honestly, you could change "Fully remote clients without a patch file" to just "Clients" but that's up to how specific you want to be.
"the" -> "their" for readability, and I think changing "such as" to a common, finite list is a better wording here, especially because the "etc." is _not_ the end of the sentence or in a parenthetical
```suggestion
Fully remote clients without a patch file may use this to "place" items onto their in-game locations, most commonly to display their names or item classifications before/upon pickup.
```

> LocationScouts can also be used to inform the server of locations the client has seen, but not checked, to create a hint as if the player had run `!hint_location` on this location, but without deducing hint points.

Reviewer 2: Reducing

Author: That's fair

Author: Although I do worry that people will think this is default behavior, so I might add "optionally" or something

> LocationScouts can also be used to inform the server of locations the client has seen, but not checked, to create a hint as if the player had run `!hint_location` on this location, but without deducting hint points.

Reviewer: This sentence is pretty lengthy and has a lot of clauses, so it can be broken up into two:
```suggestion
LocationScouts can also be used to inform the server of locations the client has seen, but not checked. This creates a hint as if the player had run `!hint_location` on a location, but without deducting hint points.
```

> This is useful in cases in which the item may appear in the game world, such as 'ledge items' in A Link to the Past. To do this, set the "create_as_hint" parameter to a non-zero value.

Reviewer: Removing the double "in", changing to a less specific article, removed modal verb for tone, italicizing game title, and converting parameter to code formatting
```suggestion
This is useful in cases where an item appears in the game world, such as 'ledge items' in _A Link to the Past_. To do this, set the `create_as_hint` parameter to a non-zero value.
```
